src/content/docs/tutorial-service-level-mgmt/intro-to-slm.mdx

```diff
@@ -0,0 +1,156 @@
+
+import apmServiceBoundary from 'images/apm_screenshot-crop_service-boundary.webp'
+
+import apmLatencyBaseline from 'images/apm_screenshot-full_latency-baseline.webp'
+
+import apmSLStartFromApm from 'images/apm_screenshot-full_sl-start-from-apm.webp'
+
+import apmAddInputServiceLevel from 'images/apm_screenshot-crop_add-input-service-level.webp'
+
+import apmSyntheticsSetup from 'images/apm_screenshot-crop_synthetics-setup.webp'
+
+import queriesnrqlTransactionListNrql from 'images/queries-nrql_screenshot-full_transaction-list-nrql.webp'
+
+import apmCapabilityFilterClause from 'images/apm_screenshot-crop_capability-filter-clause.webp'
+
+
+Within an organization, communication between stakeholders can often prove more difficult than needed. IT departments usually don't speak in terms that the business-focused parts of an organization can understand, and they in turn don't normally communicate in terms that IT teams find useful. To improve reliability, resolving this language barrier helps to prevent issues. Service level management, or the practice of standardizing data into a universal language that all parties can understand, aids in doing just that.
+
+Service level management best applies to the practice of **Uptime, performance and reliability**, but it also applies to the other practices of **Customer experience**, **Innovation and growth**, and **Operational efficiency** ([learn more about these practices](/docs/new-relic-solutions/observability-maturity/introduction)). Regardless of what area you want to improve, using SLM has two major areas of impact: business outcomes and operational outcomes.
+
+<Tabs stacked>
+	<TabsBar>
+        <TabsBarItem id="1">
+            Business outcome
+        </TabsBarItem>
+        <TabsBarItem id="2">
+            Operational outcome
+        </TabsBarItem>
+    </TabsBar>
+
+    <TabsPages>
+        <TabsPageItem id="1">
+        The required business outcome in terms of reliability focuses on reducing the number of business-impacting incidents, their duration, and the number of people involved.
+
+        1. Reduce the number of business-disrupting incidents.
+        2. Reduce mean time to resolution (MTTR).
+        3. Reduce average number of people engaged (FTEs) per severe incident.
+        </TabsPageItem>
+        <TabsPageItem id="2">
+        The required operational outcome in terms reliability focuses on communicating the health of your digital product. You measure operational success by what percentage of critical product applications you cover with your standard service levels. You can also look at the percentage of policy adoption by your primary stakeholders. You can do both by focusing on what is important to the stakeholders, ensuring simplicity, and proving the effectiveness of service level management.
+        </TabsPageItem>
+    </TabsPages>
+</Tabs>
+
+Let's take a look at some of the common terms and define our KPIs before proceeding.
+
+## Terminology [#terminology]
```

Comment:
I think you can hide some of this information, but the first three definitions might be more interesting to read in a story form. Maybe something like:

> A service level makes a behavior measurable. You measure service levels with indicators, and set them against certain objectives for expected behavior. 

* The indicators you choose to measure a behavior are called SLIs, or service level indicators. For example, an SLI might be the percentage of login API transactions that happen within 500 milliseconds. 
* Each service level needs an objective. A service level objective is a goal for a certain behavior. For example, your login API should respond within 500 milliseconds and should be error free 99.99% of the week.

By building a story around behavior with service levels, you're practicing service level management.

Reply:
Tbh I think error budget and service boundaries could go in collapsers, but with some preceding text that explains that theyre aspects of managing your SLs. You can also have a third collapser that include additional SLIs and SLOs for people to refer to if the ^ simpler explanation doesn't "click" immediately

Reply:
My main argument for this is that it seems like SLs need SLIs and they're only meaningful so long as they have SLOs. If you're missing either an SLI or SLO, you don't actually have service lvl. Separating them into collapsers make them seem more disconnected than they actually are